charts/gateway/templates/otk-init-container-configmap.yaml

```diff
@@ -0,0 +1,76 @@
+{{ if and (.Values.otk.enabled) (not .Values.database.enabled) }}
```

Comment:
do we need below condition here
{{- if not .Values.initContainers  }}

Reply:
should not matter. There maybe other init containers. We load this only for ephemeral gw + otk only